streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java

```diff
@@ -235,7 +236,7 @@ public TopologyTestDriver(final Topology topology,
     private TopologyTestDriver(final InternalTopologyBuilder builder,
                                final Properties config,
                                final long initialWallClockTimeMs) {
-        final StreamsConfig streamsConfig = new StreamsConfig(config);
+        final StreamsConfig streamsConfig = new QuietStreamsConfig(config);
```

Comment:
Could we do `final StreamsConfig streamsConfig = new StreamsConfig(config, false);` instead? I tried locally and it worked.

Reply:
That works for TopologyTestDriver, but not MockProcessorContext because that constructor is protected, and the TopologyTestDriver happens to be in the same package as StreamsConfig.

I previously had the QuietStreamsConfig used only in MockProcessorContext, but the feedback was to also use it in TopologyTestDriver.

Seeing it this way, do you think it makes sense just to use the protected constructor in TopologyTestDriver, and the QuietStreamsConfig in the MockProcessorContext?

Reply:
With the additional context,  I'm fine with the PR as is.